manifests/P/PlasmoApp/plasmo_voice/main.yaml

```diff
@@ -0,0 +1,23 @@
+manifestSpecVersion: "4.0"
```

Comment:
Add the schema `# yaml-language-server: $schema=../../../../doc/validation-schemas/4.0/main-manifest.json` before `manifestSpecVersion`, and a newline between them.